### PR TITLE
[Internal Solution] CSES/Subarray Sums I

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
AC Proofs:
C++: 
Screenshot from 2024-07-01 22-43-24
Python: 
Screenshot from 2024-07-01 22-43-07

**Review by dongliu0426 — APPROVED** [review]

File: solutions/silver/cses-1660.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+The idea is to maintain two pointers, `i` and `j`, which will initially point to the first array element. Pointer `j` will iterate through the array until it reaches `n`, and during this process, we will keep incrementing the sum by `arr[j]`. If the sum becomes greater than $x$, we will keep decrementing the sum by `arr[i]` and keep incrementing `i` until the sum becomes less than or equal to $x$. Finally, we will increment our result if the sum becomes equal to $x$.
+
+## Implementation
+**Time complexity:** $\mathcal{O} (n)$
+
+<LanguageSection>
+<CPPSection>
+```cpp
+#include <bits/stdc++.h>
+#define int long long
```
Comment (SansPapyrus683) [inline solutions/silver/cses-1660.mdx:18]: don't use macros

Comment (freakin23) [reply solutions/silver/cses-1660.mdx]: okay
removed it.

**Review by SansPapyrus683 — APPROVED** [review]
code is...acceptable

**Pull request merged**